kata_agent.go

```diff
@@ -414,6 +414,10 @@ func constraintGRPCSpec(grpcSpec *grpc.Spec) {
 	// here: https://github.com/kata-containers/agent/issues/104
 	grpcSpec.Linux.Seccomp = nil
 
+	// TODO: Remove this constraint as soon as the agent properly handles
+	// resources provided through the specification.
+	grpcSpec.Linux.Resources = nil
```

Comment:
Do we have an appropriate Issue open so we don't forget this?
Or maybe a `FIXME` in the code might suffice?

Reply:
Here it is https://github.com/kata-containers/agent/issues/144